src/dotnet/Chat.UI.Blazor/Pages/NewChatPage.razor

```diff
@@ -3,22 +3,32 @@
 
 <AuthorizeView Context="auth">
     <Authorized>
-        <div class="px-2 py-2">
-            <div class="mt-5">
-                <EditForm Model="@_model" OnValidSubmit="@OnCreate" >
-                    <DataAnnotationsValidator />
+        <Header>
+            <div class="flex flex-col">
```

Comment:
I see this piece of code is spreading around.
What if you put to MainLayout and use approach like described in https://stackoverflow.com/questions/64255443/blazor-page-elements-other-than-body